src/shared_modules/content_manager/testtool/main.cpp

```diff
@@ -38,14 +40,29 @@ static const nlohmann::json CONFIG_PARAMETERS =
                 "dataFormat": "json",
                 "contentFileName": "example.json",
                 "s3FileName": "content.filtered_little.1.xz",
-                "databasePath": "/tmp/content_updater/rocksdb",
-                "offset": 0
+                "databasePath": "/tmp/content_updater/rocksdb"
             }
         }
         )"_json;
 
+/**
+ * @brief Initialize RocksDB database. If it doesn't exist, it will be created with an initial offset of '0'.
+ *
+ */
+void initializeDatabase()
+{
+    constexpr auto INITIAL_OFFSET {"0"};
+    const auto& databasePath {CONFIG_PARAMETERS.at("configData").at("databasePath").get_ref<const std::string&>()};
+
+    Utils::RocksDBWrapper rocksDbConnector(databasePath);
+    rocksDbConnector.put(Utils::getCompactTimestamp(std::time(nullptr)), INITIAL_OFFSET);
+}
+
```

Comment:
Question:
>If it doesn't exist, it will be created with an initial offset of '0'.

> rocksDbConnector.put(Utils::getCompactTimestamp(std::time(nullptr)), INITIAL_OFFSET);

 Isn't this always setting the offset to 0?

Reply:
Yes, my bad. It's fixed now.